app/server/lib/MinIOExternalStorage.ts

```diff
@@ -111,7 +132,7 @@ export class MinIOExternalStorage implements ExternalStorage {
 
   public async hasVersioning(): Promise<Boolean> {
     const versioning = await this._s3.getBucketVersioning(this.bucket);
-    return versioning && versioning.Status === 'Enabled';
```

Comment:
I replaced `!=` with `!==`.
The addition of `versioning !== ""` is to pass type check. MinIO does not have very good documentation on what `getBucketVersioning()` might return. Based on my experiments, it could return an empty string if versioning has never been turned on for the specified bucket. I included this in the typing, and the original code won't pass type check.

Reply:
Great, makes sense, thank you.